en/sensor/ark_flow.md

```diff
@@ -0,0 +1,96 @@
+* Bosch BMI088 6-Axis IMU
+* STM32F412CEU6 MCU
+* Two Pixhawk Standard CAN Connectors
+  * 4 Pin JST GH
+* Pixhawk Standard Debug Connector
+  * 6 Pin JST SH
+* Small Form Factor
+  * 3cm x 3cm x 1.4cm
+* LED Indicators
+* USA Built
+
+## Pixhawk Setup
+
+In order to use the ARK Flow board, connect it to the CAN bus (on any Pixhawk series controller) and enable the UAVCAN driver using [UAVCAN_ENABLE](../advanced_config/parameter_reference.md#UAVCAN_ENABLE) set to 2 or 3.
+
+The steps are:
+- In *QGroundControl*:
+  - Set the parameter [UAVCAN_ENABLE](../advanced_config/parameter_reference.md#UAVCAN_ENABLE) and reboot (see [Parameters](../advanced_config/parameters.md) for information on finding and setting parameters).
+- Connect ARK Flow CAN to the Pixhawk CAN
+
+Once enabled, the module will be detected on boot.
+Flow data should be coming through at 10Hz.
+
+<span id="mounting"></span>
+### Mounting/Orientation
+
+The recommended mounting orientation is defined connectors on the board pointing towards **back of vehicle**, as shown in the following picture.
+
+![ARK Flow align with Pixhawk](../../assets/hardware/sensors/optical_flow/ark_flow_orientation.png)
+
+On **PX4**, the orientation should be set using the parameter [SENS_FLOW_ROT](../advanced_config/parameter_reference.md#SENS_FLOW_ROT).
+
+<span id="px4_configuration"></span>
+## PX4 Configuration
+
+The parameters that you may need to configure are listed below.
+
+Parameter | Description
+--- | ---
+<span id="SENS_FLOW_MAXHGT"></span>[SENS_FLOW_MAXHGT](../advanced_config/parameter_reference.md#SENS_FLOW_MAXHGT) | Maximum height above ground when reliant on optical flow.
+<span id="SENS_FLOW_MINHGT"></span>[SENS_FLOW_MINHGT](../advanced_config/parameter_reference.md#SENS_FLOW_MINHGT) | Minimum height above ground when reliant on optical flow.
+<span id="SENS_FLOW_MAXR"></span>[SENS_FLOW_MAXR](../advanced_config/parameter_reference.md#SENS_FLOW_MAXR) | Maximum angular flow rate reliably measurable by the optical flow sensor.
+<span id="SENS_FLOW_ROT"></span>[SENS_FLOW_ROT](../advanced_config/parameter_reference.md#SENS_FLOW_ROT) | Yaw rotation of the board relative to the vehicle body frame.
+
+If you're using flow with the ECL/EFK2 estimator you will also need to set the [Optical Flow > Estimators > EKF2](../sensor/optical_flow.md#ekf2) parameters.
+These enable fusing optical flow measurements for velocity calculation and also allow you to define offsets if the sensor is not centred within the vehicle.
+
+### Connecting
+
+The ARK Flow is connected to the CAN bus using a Pixhawk standard 4 pin JST GH cable.
+Multiple sensors can be connected by plugging additional sensors into the ARK Flow's second CAN connector.
+
+## Software / Build Source
+
+* PX4 Firmware (Firmware source code on Github: [PX4/PX4-Autopilot](https://github.com/PX4/PX4-Autopilot))
+
+## Build
+
+Install the [PX4 toolchain](../dev_setup/dev_env.md). 
```

Comment:
Are you saying that this requires a custom build of PX4. This isn't present in master builds we can download from QCG? If not, why are we building this.

Generally production quality uavcan stuff should not need firmware update right?

Essentially I am not all that familiar with UAVCAN so need clarity on the purpose here.

Reply:
It does not require a custom build of PX4 on the flight controller side. Since ARK Flow runs PX4 cannode, as drivers and PX4 are improved, someone may want to update their unit with new firmware. See the cuav can gps in the px4 tree. 
The ARK Flow binaries will be in the release assets as well.